docs-ref-conceptual/update-azure-cli.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Update the Azure CLI
+description: Reference for the update of Azure CLI
+author: dbradish-microsoft
+ms.author: dbradish
+manager: barbkess
+ms.date: 10/19/2020
+ms.topic: conceptual
+ms.service: azure-cli
+ms.devlang: azurecli 
+ms.custom: devx-track-azurecli
+---
+
+# Update the Azure CLI
+
+You can rely on package managers to update Azure CLI on Windows, macOS and Linux environments (see the `Update` section in each platform-specific install instruction). The CLI also provides in-tool commands to upgrade manually or automatically.
```

Comment:
You can rely on package managers to update a local install of the Azure CLI on Windows, macOS...